fdbclient/Schemas.cpp

```diff
@@ -181,6 +181,18 @@ const KeyRef JSONSchemas::statusSchema = LiteralStringRef(R"statusSchema(
                      "p99":0.0,
                      "p99.9":0.0
                   },
+                  "commit_batching_window_size":{
```

Comment:
Since you modified the schema, I suppose `documentation/sphinx/source/mr-status-json-schemas.rst.inc` should be updated as well.

In addition, `documentation/sphinx/source/release-notes/release-notes-700.rst` may need an update on the `Status` section.

Reply:
> Since you modified the schema, I suppose `documentation/sphinx/source/mr-status-json-schemas.rst.inc` should be updated as well.
> 
> In addition, `documentation/sphinx/source/release-notes/release-notes-700.rst` may need an update on the `Status` section.

Done. https://github.com/apple/foundationdb/pull/4748 .